### PR TITLE
Update style GitHub actions

### PR DESCRIPTION
**Pull request by Yozhef**

### CODE REVIEW

File: Tests/BundleInMemoryTest.php [hunk 1]
```diff
@@ -12,16 +12,16 @@ class BundleInMemoryTest extends WebTestCase
...
         return AppKernel::class;
     }
 
-    protected static function createKernel(array $options = array())
+    protected static function createKernel(array $options = [])
     {
         $class = self::getKernelClass();
 
-        if (!isset($options['test_case'])) {
+        if (! isset($options['test_case'])) {
```
Comment (denisvmedia) [inline Tests/BundleInMemoryTest.php:24]: This change seems to go against symfony standards. https://symfony.com/doc/current/contributing/code/standards.html

File: Tests/BundleRedisTest.php [hunk 2]
```diff
@@ -36,12 +36,12 @@ protected static function createKernel(array $options = array())
 
     private static function getVarDir(): string
     {
-        return 'FB'.substr(strrchr(static::class, '\\'), 1);
+        return 'FB' . substr(strrchr(static::class, '\\'), 1);
```
Comment (denisvmedia) [inline Tests/BundleRedisTest.php:39]: This change goes against symfony coding standards.

**Comment by Yozhef** [comment]
@denisvmedia thanks for the review I'm fixed all your comments.

**Comment by Yozhef** [comment]
@denisvmedia can you review this and release a new version?

**Comment by denisvmedia** [comment]
I implemented code style checks the way how we do it in our private repositories. So, I suggest we close this MR.

**Pull request closed without merging**